api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCalculation.java

```diff
@@ -1,17 +1,39 @@
 package uk.gov.dhsc.htbhf.claimant.service.payments;
 
 import lombok.Builder;
-import lombok.Data;
+import lombok.Value;
 import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycleStatus;
 
+import java.time.LocalDateTime;
+
+import static uk.gov.dhsc.htbhf.claimant.entity.PaymentCycleStatus.BALANCE_TOO_HIGH_FOR_PAYMENT;
+
 /**
  * The result of a payment calculation from {@link PaymentCalculator}.
  */
-@Data
+@Value
 @Builder
 public class PaymentCalculation {
 
     private PaymentCycleStatus paymentCycleStatus;
-
     private int paymentAmount;
+    private int availableBalanceInPence;
+    private LocalDateTime balanceTimestamp;
+
+    public static PaymentCalculation aFullPaymentCalculation(int paymentAmount) {
+        return PaymentCalculation.builder()
+                .paymentAmount(paymentAmount)
+                .paymentCycleStatus(PaymentCycleStatus.FULL_PAYMENT_MADE)
```

Comment:
You've used a static import for the one below so let's use it here too.

Reply:
Done.